app/assets/stylesheets/overlays/_paper_submit_overlay.scss

```diff
@@ -1,11 +1,35 @@
 .paper-submit-overlay {
   text-align: center;
 
-  h1 {
-    margin-bottom: 0.5em;
+  .overlay-title-text h1 {
+    padding-top: 20px;
+    padding-bottom: 40px;
+  }
+  h5 {
+    opacity: 0.5;
+  }
+  .submit-action-buttons {
+    display: block;
+    margin: auto;
+
+    button {
+      text-align: center; // override
+      display: block;
+      margin: 40px auto 0 auto;
+      &:last-child {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3